src/Filter.js

```diff
@@ -75,16 +76,18 @@ export default class Filter extends Component {
         <h3>Distance From Denver</h3>
         <div className="filter-range-section">
           <p>{this.state.setMax} miles away</p>
+          <img className="hiker-image" src={Hiker} alt="hiker"/> 
```

Comment:
Nice that you created a separate span for the checkbox. I was thinking we could make it bigger. :)